<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Browser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableView">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboSort"/>
        </item>
        <item>
         <widget class="QPushButton" name="submitButton">
          <property name="text">
           <string>Submit</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="revertButton">
          <property name="text">
           <string>Revert</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionAdd_Connection"/>
   </widget>
   <widget class="QMenu" name="menuProperties">
    <property name="title">
     <string>&amp;Properties</string>
    </property>
    <addaction name="actionHide_vertical_headers"/>
   </widget>
   <widget class="QMenu" name="menuSortings">
    <property name="title">
     <string>&amp;Sortings</string>
    </property>
    <addaction name="actionQMap"/>
    <addaction name="actionQuickSort"/>
    <addaction name="actionHeapSort"/>
    <addaction name="actionStableSort"/>
    <addaction name="actionSimpleSort"/>
    <addaction name="actionTimSort"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProperties"/>
   <addaction name="menuSortings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Connection">
   <property name="text">
    <string>Add &amp;Connection...</string>
   </property>
  </action>
  <action name="actionHide_vertical_headers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hide vertical headers</string>
   </property>
  </action>
  <action name="actionQMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>QMap</string>
   </property>
  </action>
  <action name="actionQuickSort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quick Sort</string>
   </property>
  </action>
  <action name="actionHeapSort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Heap Sort</string>
   </property>
  </action>
  <action name="actionStableSort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stable Sort</string>
   </property>
  </action>
  <action name="actionSimpleSort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simple Sort</string>
   </property>
  </action>
  <action name="actionTimSort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tim Sort</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
